eox_nelp/openedx_filters/tests/xapi/tests_filters.py

```diff
@@ -0,0 +1,70 @@
+"""This file contains all the test for the filters.py file.
+
+Classes:
+    XApiActorFilterTestCase: Tests cases for XApiActorFilter filter class.
+"""
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from tincan import Agent
+
+from eox_nelp.openedx_filters.xapi.filters import XApiActorFilter
+
+User = get_user_model()
+
+
+class XApiActorFilterTestCase(TestCase):
+    """Test class for XApiActorFilter filter class."""
+
+    def setUp(self):
+        """Setup common conditions for every test case"""
+        self.filter = XApiActorFilter(
+            filter_type="event_routing_backends.processors.xapi.transformer.xapi_transformer.get_actor",
+            running_pipeline=["eox_nelp.openedx_filters.xapi.filters.XApiActorFilter"],
+        )
+        self.username = "xapi"
+        self.email = "xapi@example.com"
+        User.objects.update_or_create(username=self.username, email=self.email)
+
+    def test_user_does_not_exist(self):
+        """ Test case when the user is not found by the email.
+
+        Expected behavior:
+            - Returned value is the same as the given value.
```

Comment:
I thought that in this test would not have actor, but in this case is not modify

Reply:
The method uses the given value to find a user, if the user is not found the method won't alter the result or will act as identity method